adcc/ExcitedStates.py

```diff
@@ -338,6 +375,16 @@ def describe(self, oscillator_strengths=True, rotatory_strengths=False,
             text += body.format(i=i, ene=self.excitation_energy[i],
                                 ev=self.excitation_energy[i] * eV, **fields)
         text += separator + "\n"
+        if len(self._excitation_energy_corrections):
+            head_corr = "|  excitation energy corrections included:"
+            text += head_corr
+            nspace = len(separator) - len(head_corr) - 1
+            text += nspace * " " + "|\n"
```

Comment:
Could you show an example output? I'm curious what it looks like.

Reply:
Will show up in the docs 😄

Reply:
Ok. I would write "Excitation energy includes these corrections:". I think that's more clear.

Reply:
Done, also updated in docs example.